aggregate-mappings.json

```
@@ -7,6 +7,7 @@
         "sda/cmd/verify/verify.md": "docs/services/verify.md",
         "sda/cmd/s3inbox/s3inbox.md": "docs/services/s3inbox.md",
         "sda/cmd/syncapi/syncapi.md": "docs/services/syncapi.md",
-        "sda/sda.md": "docs/services/sda.md"
+        "sda/sda.md": "docs/services/sda.md",
+        "GETTINGSTARTED.md": "docs/sda-dev-test-doc.md"
```

```suggestion
        "GETTINGSTARTED.md": "docs/guides/sda-dev-test-doc.md"
```
Or am I missing something?